### ISSUE TITLE
SevTech server disappeared

### ISSUE THREAD

**Issue by lmdpocus** [hexparrot/mineos-node#321]
I'm trying to get the docker to work with SevTech (1.2.3 Hotfix 1) running.

It was working and running, could join and play, but it did vanish within the web user interface, which makes it hard to manage, even when you still can join and play on the server.

Refreshing the "Refresh Server List" is not helping either with it. 

Running git commit: d743d22 on Docker 

Config of the server.config file;
[java]
java_xmx=16384
java_xms=4096
jarfile=forge-1.12.2-14.23.5.2838-universal.jar
java_tweaks="-XX:+UseG1GC -Dsun.rmi.dgc.server.gcInterval=2147483646 -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -Dfml.readTimeout=180"
jar_args=nogui

[onreboot]
start=false

[minecraft]
profile=1.12.2

Other information:
Minecraft instance Latest.log:
[latest.log](https://github.com/hexparrot/mineos-node/files/4009508/latest.log)
MineOS log(IP's masked out):
[mineos.log](https://github.com/hexparrot/mineos-node/files/4009510/mineos.log)

If more information is needed, let me know.

**Comment by lmdpocus** [comment]
Some more information, when i say "onreboot" to true, it starts the server
![image](https://user-images.githubusercontent.com/1772909/71578636-48832900-2af9-11ea-852c-0fffadcff1e5.png)

But in the webinterface it is not visible, even added a "test server" to see if there was an other problem.
![image](https://user-images.githubusercontent.com/1772909/71578681-694b7e80-2af9-11ea-9e8d-d0c939de56b2.png)

**Comment by Simmons928** [comment]
Not sure if this is going to be helpful or not. 
I have been troubleshooting a similar thing on my server running in a docker on unraid. 

I had similar symptoms to you. 
I had two servers, both would start and I could join them, but only one would show in the interface (whichever was first alphabetically seemed to determine which it was.)
the tail would only update if I refreshed the webpage. 

I tracked this down on my server to a permissions issues. 
How this started for me:
-I rebuilt my array to get rid of some old drives and reset all file permissions to nobody nobody 777.
-I did not think permissions would be the issue as it was set to 777

I stopped the docker, renamed the server directory, and started it back up. 
The docker began working normally. 
I moved the servers back in, restarted the docker and it stopped working again. 

The only difference was the permissions between the new files and the old files so I tried that and that did the trick. 

I tried setting the setting back to nobody nobody 777 and the issue started back up so there is something permissions related that seems to cause an issue, at least with a docker in unraid.

I had to set the permissions to all of the files under the mineos server files to the following (and I did this using WinSCP), which is the default setting for the permissions.
Group 1000
User 1000
octal 0664
